toolchain/install/install_filegroups.bzl

```
@@ -0,0 +1,132 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""Rules for constructing install information."""
+
+load("@rules_pkg//pkg:mappings.bzl", "pkg_attributes", "pkg_filegroup", "pkg_files", "pkg_mklink", "strip_prefix")
+load("symlink_helpers.bzl", "symlink_file", "symlink_filegroup")
+
+def install_filegroup(name, filegroup_target):
+    """Adds a filegroup for install.
+
+    Used in the `install_dirs` dict.
+
+    Args:
+      name: The base directory for the filegroup.
+      filegroup_target: A relative path for the symlink.
+    """
+    return {
+        "filegroup": filegroup_target,
+        "for_driver": False,
+        "name": name,
+    }
+
+def install_symlink(name, symlink_to):
+    """Adds a symlink for install.
+
+    Used in the `install_dirs` dict.
+
+    Args:
+      name: The filename to use.
+      symlink_to: A relative path for the symlink.
+    """
+    return {
+        "for_driver": False,
+        "name": name,
+        "symlink": symlink_to,
+    }
+
+def install_target(name, target, executable = False, for_driver = False):
```

What do you think about s/`for_driver`/`is_driver`/ ? That would read a bit better to me I think...

```suggestion
def install_target(name, target, executable = False, is_driver = False):
```

---

Done

---

Done